src/app/portlets/dot-edit-content/dot-edit-content.component.ts

```diff
@@ -40,6 +42,7 @@ export class DotEditContentComponent implements OnInit {
     isModelUpdated = false;
     page: DotRenderedPage;
     pageWorkFlows: Observable<Workflow[]>;
+    device: Device;
```

Comment:
why do we need this attribue?

Reply:
to set the dimensions of `dot-edit__page-wrapper`